kubeflow/trainer/rhai/transformers.py

```
@@ -323,9 +333,297 @@ def checkpoint_in_progress(self):
+                        flush=True,
+                    )
+                except Exception as e:
+                    raise RuntimeError(
+                        f"Failed to check this node has access to the storage path: "
+                        f"'{cloud_remote_storage_uri}'. Error: {e}. "
+                        f"If using self-signed certificates, "
+                        f"set verify_cloud_storage_ssl=False. "
+                        f"If experiencing permission issues, check you have read and "
+                        f"write permissions to '{cloud_remote_storage_uri}'. "
+                        f"This check can be disabled by setting verify_cloud_storage_access=False."
+                    ) from e
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)